### ISSUE TITLE
it breaks when loading remote images

### ISSUE THREAD

**Issue by sanabel-al-firdaws** [HiDeoo/starlight-image-zoom#14]
### Describe the bug

![nice image](https://d2.atlas.lucas.tools/?script=FMwxDsIwDAXQPaf4Wyd6gAyMSLCWC7jUIhE0rmzTNjk9yvykd0Y8E_lgeIvDBTPDE2Nmc2xZyRn3YWfwzgpjLuHE5YoaasQkyI5Dft-l0zqOY6hdW8SDXh9MG6nKMVg_V4Y5qVtoETdaGLm4IInmJiX8AwAA__8%3D&)

this works when disabling the plugin but when the plugin is on the image disapears but i can see it in dev tools

### To Reproduce

1. enable the plugin
2. copy paste this and add it to any page
```
![nice image](https://d2.atlas.lucas.tools/?script=FMwxDsIwDAXQPaf4Wyd6gAyMSLCWC7jUIhE0rmzTNjk9yvykd0Y8E_lgeIvDBTPDE2Nmc2xZyRn3YWfwzgpjLuHE5YoaasQkyI5Dft-l0zqOY6hdW8SDXh9MG6nKMVg_V4Y5qVtoETdaGLm4IInmJiX8AwAA__8%3D&)


```

### Expected behavior

showing the image on the page

### How often does this bug happen?

Every time

**Comment by HiDeoo** [comment]
Thanks for the report.

I assume that the issue is due to the fact that the image is a remote SVG and the wrapping web component is [using](https://github.com/HiDeoo/starlight-image-zoom/blob/3b629e622548e42cb91c35e8cac4b1c5ce2fc651/packages/starlight-image-zoom/components/ImageZoom.astro#L38) `display: inline-block`. This property is mostly used to properly display the zoom button at the top right of the image.

Not quite sure what would be the proper way to fix this without thinking and playing around with it for a bit. I'm also open to suggestions or pull requests in the meantime if someone has ideas on how to fix this.

**Comment by HiDeoo** [comment]
This should now be fixed in version [`0.7.0`](https://github.com/HiDeoo/starlight-image-zoom/releases/tag/v0.7.0) that has been just published 🎉
